README.md

```diff
@@ -1,2 +1,36 @@
 # hw01
 Repository for homework 1
+
+Text added through RStudio
+
+A brief biography as a part of the homework:
+
+## Zahra Amalia Syarifah
+I was born in Indonesia and grew up there. However, shortly before my high school graduation I moved to the UK. *So I **never** actually completed high school*. I went straight to a college in Southeast England and moved to Scotland for my undergraduate. Now, I am completing my Master in the States.
```

Comment:
The use of * to emphasize bold or italics is not exactly right. It begins correctly with "So I never..." but it doesn't go to the end of the intended command. I'm not sure how to fix this, but you could try using the _ command for italics and the * for bold so there is no confusion in the line of code.